analysis/topEFT/make_cr_and_sr_plots.py

```diff
@@ -435,7 +435,7 @@ def make_all_cr_plots(dict_of_hists,year,unit_norm_bool,save_dir_path):
     eft_sow_scale_dict = yt.get_eft_sow_scale_dict(dict_of_hists["SumOfEFTweights"])
 
     # Loop over hists and make plots
-    skip_lst = ["SumOfEFTweights"] # Skip this hist
+    skip_lst = ["SumOfEFTweights","ptz"] # Skip this hist
```

Comment:
Do we want to skip `ptz`, or was this for a test?

Reply:
Oops. Sorry about that. I was having trouble making plots specifically for ptz CRs. Which is why I included it in that list.